### PR TITLE
llama: init at 1.0.1

### PR DESCRIPTION
**Pull request by portothree**
###### Motivation for this change
https://github.com/antonmedv/llama

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/llama/default.nix
Comment (legendofmiracles) [inline pkgs/applications/misc/llama/default.nix:20]:
```suggestion
```

File: pkgs/applications/misc/llama/default.nix
Comment (legendofmiracles) [inline pkgs/applications/misc/llama/default.nix:16]: Missing maintainer and license

**Review by legendofmiracles — CHANGES_REQUESTED** [review]

**Comment by legendofmiracles** [comment]
All package related commit should also be squashed into `llama: init at 1.0.1`

**Comment by portothree** [comment]
@legendofmiracles Updated accordingly, mid re-reviewing? Thank you

File: pkgs/applications/misc/llama/default.nix
Comment (legendofmiracles) [inline pkgs/applications/misc/llama/default.nix:19]:
```suggestion
    license = licenses.mit;
```

**Comment by portothree** [comment]
@legendofmiracles could you re-review please? :pray:

**Comment by legendofmiracles** [comment]
@ofborg build llama

File: maintainers/maintainer-list.nix
Comment (legendofmiracles) [inline maintainers/maintainer-list.nix:13416]: Why are you adding two maintainers? 

They should also each be added in their own commit: `maintainers: add <name>`

After that everything LGTM

Comment (portothree) [reply maintainers/maintainer-list.nix:13416]: The contributing doc wasn't very clear if maintainers should be project owner/maintainer or just someone to maintain the nix module, so I have added both. 
It's means that I have to open a different PR to include maintainers or just pick the commit outside of `llama: init at 1.0.1`?

Comment (legendofmiracles) [reply maintainers/maintainer-list.nix:13416]: Ah no, only the maintainer of the nix code should be added. 

A different commit is fine.

Comment (portothree) [reply maintainers/maintainer-list.nix]: Ok, updated

**Comment by legendofmiracles** [comment]
In it's current form this does not build `go: inconsistent vendoring in /build/source:`.

Upstream just released a new update, it also fixes the build failure, I've written a patch here: http://0x0.st/-F7-.txt

**Comment by portothree** [comment]
@legendofmiracles sorry for the direct ping again, but do you mind to re-review this? Thank you

**Review by legendofmiracles — APPROVED** [review]

**Comment by legendofmiracles** [comment]
Can you please resolve the merge conflict? After that I could go ahead and merge.

**Comment by legendofmiracles** [comment]
Sorry, can you squash the merge commit? While you're at it, it would also be great if the maintainers commit could be before the package one
That's easily done by only reordering the lines in `git rebase`

**Comment by legendofmiracles** [comment]
Whoops my bad 😅

**Comment by legendofmiracles** [comment]
This is entirely my fault, I was trying to fix your mistakes... Could you open a new PR with your changes? If you don't have them anymore then apply this: http://0x0.st/oiez.txt

**Pull request closed without merging**
